docker-compose.yml

```diff
@@ -32,6 +32,9 @@ services:
       - CELERY_BROKER_URL=amqp://rabbitmq
     networks:
       - openimis-net
+    volumes:
+      - "${BE_CONF_PATH}:/openimis-be/openimis.json"
+      - "${DEV_CONF_PATH:-/dev/null}:/openimis-dev"
```

Comment:
Does it work on windows ?

Reply:
you are right, according to https://stackoverflow.com/questions/313111/is-there-a-dev-null-on-windows it should be 'nul', for windows